src/app/layout.tsx

```diff
@@ -1,8 +1,11 @@
 import "./globals.css";
 import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+import { Poppins } from "next/font/google";
 
-const inter = Inter({ subsets: ["latin"] });
+const font = Poppins({
+  weight: ["400", "700"],
+  subsets: ["latin"],
+});
```

Comment:
[Question]: Any reason why we're only importing *normal* and *bold* weights for Poppins 🤔 How do you feel about also modifying the [weights in the tailwind theme](https://tailwindcss.com/docs/font-weight#using-custom-values) to reflect this? If you think its pointless then no biggie, we can keep the tailwind theme as it is and just ensure we only use `font-normal` and `font-bold` 😎.

Reply:
Each font-weight requires a separate font definition to use; including all of them by default is generally not recommended. It's a good callout though that we'll need to watch for uses of other font weights accidentally (made much easier by tailwind) but there are good reasons to do that anyways.

Reply:
Re-reading the link you sent; I see your idea was to remove these from Tailwind entirely- that's a great idea and I'll definitely give that a try.

Reply:
From discussion on Discord, we'll leave this alone as it doesn't have the desired effect:

1. Limiting the available fontWeights doesn't protect against using them; the classes are just undefined.
2. The config defines a global theme which isn't specific to the font - so you may have another font which supports 500 and would want access to font-medium there.
3. We seem to be using CSS Modules instead of Tailwind anyways, so I'm inclined to stick with that anyways rather than try to customize this.